lib/libs/opensearch-lib.ts

```diff
@@ -168,15 +167,11 @@ export async function mapRole(
 
 export async function search(host: string, index: opensearch.Index, query: any) {
   client = client || (await getClient(host));
-  try {
```

Comment:
removed try/catch so that errors can be caught and handled by the calling function